src/@types/MetaData.d.ts

```diff
@@ -38,6 +38,7 @@ export interface MetadataPublishFormDataset {
   // ---- optional fields ----
   tags?: string
   links?: string | EditableMetadataLinks[]
+  customProvider?: string
```

Comment:
let's be consistent across projects with variables and name this variable `providerUri` everywhere instead of `customProvider` (the component name is fine though)

Reply:
This has been done